source/data/History.rst

```diff
@@ -5,41 +5,40 @@
 History
 -------
 
-We need the `/history`_ endpoint to:
+TradingView needs the `/history`_ endpoint to:
 
 * fill the database with deep history,
-* compensate data from the `/streaming`_ in case of problems.
+* compensate data from `/streaming`_ in case of problems.
 
-After initial filling of the database we make regular requests to `/history`_ endpoint to keep the data relevant. Our 
-data feed requests 1-minute bars for the whole day per request. Requests are made sequentially from the current time 
-to the past. 
+After initial filling of the database, TradingView makes regular requests to `/history`_ endpoint to keep the data relevant.
+TradingView data feed requests 1-minute bars for the whole day per request.
+Requests are made sequentially from the current time to the past. 
 
-If there is no data in the requested and previous time periods then you should set the status code to ``no_data``. The
-API should respond with an empty response in case of requesting the range containing no historical data.
+Data requirements
+..................
 
-We will request `/history`_ until the date that the broker reported in the **Data requirements from**. Without this
-date, we will request a history up to 1800 year.
+Your data should meet the following requirements:
 
-.. code-block:: json
+- The data must be real, from the production environment preferably. Otherwise, integration tests will fail.
+- Real-time data obtained from the `/streaming`_ endpoint must match the historical data obtained from `/history`_. The allowed count of mismatched bars (candles) must not exceed 5% for frequently traded symbols. Otherwise, integration into TradingView is not possible.
```

Comment:
I'd say `number/amount` instead of `count`. Also the words `allowed` and `must not` are kind of repeating. I suggest to remove one of them.
```suggestion
- Real-time data obtained from the `/streaming`_ endpoint must match the historical data obtained from `/history`_. The number of mismatched bars (candles) must not exceed 5% for frequently traded symbols. Otherwise, integration into TradingView is not possible.
```
or
```suggestion
- Real-time data obtained from the `/streaming`_ endpoint must match the historical data obtained from `/history`_. The allowed number of mismatched bars (candles) does not exceed 5% for frequently traded symbols. Otherwise, integration into TradingView is not possible.
```